app/routes/questionnaire.py

```diff
@@ -340,6 +340,26 @@ def send_confirmation_email(session_store, schema):
     )
 
 
+@post_submission_blueprint.route("confirmation-email/confirm", methods=["GET", "POST"])
+@with_schema
+@with_session_store
+def confirm_confirmation_email(session_store, schema):
+    confirm_email = ConfirmEmail(
```

Comment:
I find the confirm/confirmation naming pattern confusing, but am restricted with the current design